openeo/imagecollection.py

```diff
@@ -52,7 +52,14 @@ def bands(self) -> List[Band]:
 
     def _get_bands(self) -> List[Band]:
         # TODO refactor this specification specific processing away in a subclass?
-        # First try `properties/eo:bands`
+        # First try `summaries/eo:bands`
+        eo_bands = self.get('summaries', 'eo:bands')
```

Comment:
ah ok, I didn't know about this `summaries` field as replacement for the `properties` field.
Maybe we can drop support for `properties` in favor of `summaries`. I have no idea if it is necessary to keep supporting that

Reply:
Yes, also "cube:dimensions" is now one layer up in the dictionary, I updated that in my last commit.